client/src/main/scala/com/github/dapperware/slack/models/Message.scala

```diff
@@ -4,19 +4,69 @@ import io.circe.Codec
 import io.circe.generic.semiauto._
 
 case class Message(
-  user: String,
-  text: String,
-  thread_ts: String,
-  ts: String,
+  user: Option[Int] = None,
+  text: Option[Int] = None,
+  thread_ts: Option[Int] = None,
+  ts: Option[String] = None,
+  team: Option[String] = None,
+  hidden: Option[Boolean] = None,
+  deleted_ts: Option[String] = None,
+  event_ts: Option[String] = None,
+  bot_id: Option[String] = None,
+  username: Option[String] = None,
+  icons: Option[Icon] = None,
+  bot_profile: Option[BotProfile] = None,
+  inviter: Option[String] = None,
+  topic: Option[String] = None,
+  purpose: Option[String] = None,
+  name: Option[String] = None,
+  old_name: Option[String] = None,
+  members: Option[String] = None,
+  parent_user_id: Option[String] = None,
+  files: Option[List[SlackFile]] = None,
+  upload: Option[Boolean] = None,
+  comment: Option[SlackComment] = None,
+  item_type: Option[String] = None,
   reply_count: Option[Int] = None,
   replies: Option[List[Reply]] = None,
-  parent_user_id: Option[String] = None,
+  reply_to: Option[Int] = None,
```

Comment:
I think this should be a string as well

Reply:
Not according to https://github.com/slack-go/slack/blob/0c3c3c75269d9fdd0e53923e26e97b1078352680/messages.go#L16